### ISSUE TITLE
Large Home Range Estimate for Individuals with Low Number of Waypoints

### ISSUE THREAD

**Issue by ofmerrit** [ctmm-initiative/ctmmweb#133]
I have been estimating home ranges for Hermit Thrushes, and almost all of them have 15 or more waypoints associated with them. One individual, however, has 7 waypoints, and the home range estimate ctmmweb gives is 27 square hectares. This is a dramatic difference between the others, who have estimated home ranges between 4 and 10 square hectares.

My question is: is the model over-extrapolating since the individual only has 7 waypoints? The individual has one point that was not at first regarded as an outlier but may be. It's not, however, an outlier to that would make a 27 square hectare home range estimate.

**Comment by chfleming** [comment]
The small effective sample size bias in pHREML-AKDE is predominantly negative, so it's probably not because there are only 7 locations. I would consider:

1. How wide are the CIs? The point estimate may be 27 ha, but if the CI is 2-50 ha, then that's still consistent with the other individuals.
2. Are the times correct and/or are there any locations close together in time, so that they would need a location error model.
3. Was the individual range resident? This is harder to say with 7 locations, but, for instance, 7 locations traveling north would not be resident and would yield a large range estimate with ~1 degree of freedom.

**Comment by ofmerrit** [comment]
The CIs are 3.72 to 73.73. On another individual where we only have 10 waypoints, his home range is crazy: 8054 ha (with CI from 0 to 7497937).

How close together in time would a location error model happen? The individuals don't have waypoints collected within a half hour of each other. 

All of the individuals are resident.

**Comment by chfleming** [comment]
Yeah, those estimates are consistent, but they have extremely small effective sample sizes. That could be from a fundamental limitation of the data, or it could be AIC being not very good at selecting a model, given small samples of autocorrelated data. You might try a more stringent model selection criterion, or, manually choosing `isotropic=TRUE` if it makes sense for the data. I've run into similar issues with Argos data on range resident birds, where the effective sample sizes were tiny, but AIC/BIC just wouldn't select a more parsimonious model with more sensible outputs.

Unmodeled error can cause issues if the location error is comparable to or larger than how far the animal tends to move within the sampling interval. So, with GPS data, if you have a burst of 1Hz data, then the 10 meter location error can far exceed the regular animal movement, and then without an error model that variance gets modeled as movement, throwing the estimates way off.
